include/sensors/core/tasks/capacitive_sensor_callbacks.hpp

```diff
@@ -72,10 +73,15 @@ struct ReadCapacitanceCallback {
         }
         auto capacitance = convert_capacitance(measurement, number_of_reads,
                                                current_offset_pf);
-        auto message = can_messages::ReadFromSensorResponse{
-            .sensor = SensorType::capacitive,
-            .sensor_data = convert_to_fixed_point(capacitance, 15)};
-        can_client.send_can_message(can_ids::NodeId::host, message);
+        if (utils::tag_in_token(m.id.token,
+                                utils::ResponseTag::IS_THRESHOLD_SENSE)) {
+            set_threshold(capacitance + 1);
+        } else {
+            auto message = can_messages::ReadFromSensorResponse{
+                .sensor = SensorType::capacitive,
+                .sensor_data = convert_to_fixed_point(capacitance, 15)};
```

Comment:
Should this be a `q15_16`? I might've messed up the fixed point location before.

Reply:
It matches up with the python side, so I'd like to keep it for now at least